pkg/controller/volume/scheduling/scheduler_binder_test.go

```diff
@@ -269,18 +266,20 @@ func (env *testEnv) updateVolumes(t *testing.T, pvs []*v1.PersistentVolume, wait
 	}
 	if waitCache {
 		wait.Poll(100*time.Millisecond, 3*time.Second, func() (bool, error) {
-			for _, pv := range pvs {
```

Comment:
why is the check changed from checking all pvs to only checking the first?

Reply:
Hi @msau42, actually the loop always returns after the first successful execution for pvs. That's why it changed to check only the first one to fix the static check error.

Reply:
I think that was a bug in the original test then.  It should be checking all pvs

Reply:
sure, i will update it as so. thanks.